### PR TITLE
Update tests for v0.5

### PR DESCRIPTION
**Pull request by joehuchette**

### CODE REVIEW

File: src/JuMP.jl [hunk 1]
```diff
@@ -19,6 +19,7 @@ import MathProgBase
 using Calculus
 using ReverseDiffSparse
 using ForwardDiff
+using Compat
```
Comment (mlubin) [inline src/JuMP.jl]: Needs an entry in ``REQUIRE``

**Comment by mlubin** [comment]
Cool, hopefully with this we'll be back to hanging on travis in GLPK...

**Comment by mlubin** [comment]
Closes https://github.com/JuliaOpt/JuMP.jl/issues/700

**Comment by joehuchette** [comment]
Still failing on v0.5 because DataStructures needs a new tagged release...

**Comment by mlubin** [comment]
I'm okay with merging if it passes locally.

**Comment by mlubin** [comment]
Squash?

**Pull request merged**